### PR TITLE
docs: add backup success kibana queries

### PR DESCRIPTION
**Pull request by WalterMoar**

### CODE REVIEW

File: docs/Developer/DevOps/Helpful-Kibana-Queries.md [hunk 1]
```diff
@@ -36,5 +36,13 @@ kubernetes.namespace_labels.environment:prod AND /error|warn/ AND NOT message:"/
 kubernetes.namespace_labels.environment:prod AND kubernetes.container_name.raw:postgresql AND (message:"\"level\":\"error\"" OR message:"\"level\":\"warn\"")
 ```
 
+#### Database Backups:
+
+We have five databases and the backups run every night. So this query should have 10 hits per day (or 70 hits if your Time Range is `Last 7 days`).
+
+```
+message:"Successfully backed up" OR message:"Successfully verified backup"
```
Comment (bhuvan-aot) [inline docs/Developer/DevOps/Helpful-Kibana-Queries.md:44]: @WalterMoar I see 6 hits for this query, am I missing something?

Comment (WalterMoar) [reply docs/Developer/DevOps/Helpful-Kibana-Queries.md:44]: Hmm, 6 for the last 24 hours? Should be 10!
image

Comment (bhuvan-aot) [reply docs/Developer/DevOps/Helpful-Kibana-Queries.md:44]: looks like I do not have access to logs of fider and redash :)

**Review by bhuvan-aot — APPROVED** [review]

File: docs/Developer/DevOps/Helpful-Kibana-Queries.md (hunk 1 above)
Comment (WalterMoar) [reply docs/Developer/DevOps/Helpful-Kibana-Queries.md:44]: You do now! :)

Comment (bhuvan-aot) [reply docs/Developer/DevOps/Helpful-Kibana-Queries.md:44]: Excellent! thank you.
image

**Pull request merged**